Feat: new style

bbucsek opened the pull request:
# add new style to the application

re-design the application style based on the figma doc.

bbucsek committed:
feat: change sidebar style

bbucsek committed:
feat: add style changes

bbucsek committed:
feat: change stzle

bbucsek committed:
feat: add icons to the playlistpage

bbucsek committed:
feat: change form styles

bbucsek committed:
feat: change login style

KlaraHajdu reviewed (changes requested):
It looks much nicer than before, great! :)

A few things:

- would be good to have tooltips for the icons, as they are not clear at once, what they mean
- we do not need a play icon, if there are no songs in the playlist, it does not do anything
- we do not need the unfollow icon for our own playlist, it does not do anything
- my preference would be not adding the follow icon for others' playlists, but it is up to you
- if the content fits, we do not need the vertical scrollbar
- we do not need the horizontal scrollbar 
- on the songitem, the "added by..." and "votes" texts are far too small, now hardly legible. 
-  when I create a new playlist, the page looks and behaves strangely.
- why deleting the owner of the playlists that I follow? Now it does not turn out, who owns the playlists, and there can be several playlists of the same title as well, so especially difficult to tell them apart. My preference would be to show the owner on the sidebar (maybe with smaller fonts, if better), but if not, then on the playlist page.

bbucsek commented:
- tooltips: in a different pr (im not sure i want to include a package for that, but if i create a new one, that might take some time)
- play icon fixed
- unfollow icon fixed
- follow icon: lets keep it untill we rework the invite method
- scrollbar: fixed
- font size: i increase a bit
- strange behaviour: fixed
- i dont think that this is the right place to show who is the owner. the sidebar has limited space, and with a long playlistname with a username (which is often a full name) it will be too long. but we should show in the playlistpage, who created the playlist and who is following that.

bbucsek committed:
fix: add requested changes
